Devws for most target limited co 144

adurosmart opened the pull request:
1, In the last version，After the button 2 or 3  is pressed, LEVEL_CONTROL_CLUSTER and ADUROSMART_SPECIFIC_CLUSTER two raw data are reported at the same time, and LEVEL_CONTROL_CLUSTER will be reported multiple times,
2,Now, the analysis of the button data of LEVEL_CONTROL_CLUSTER on the button 2 or 3 is removed, and only  parsed the data of ADUROSMART_SPECIFIC_CLUSTER on the button 2 or 3 , so that other engineer can get the single push event result of triggered by the once press button 2 or 3.
Thank you for review.

adurosmart committed:
Update zigbee-multi-button.groovy

@tpmanley
Thank you for your suggestions, (https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/13835#issuecomment-587027214), I followed your suggestions to update the code.
Thank you for reviewing the code again.

adurosmart committed:
Fixed bugs and code style issues

@greens Thank you review.
Fixed bugs and code style issues.

Commit:
Update zigbee-multi-button.groovy

1、In the last version，After the 2 or 3 button are pressed, LEVEL_CONTROL_CLUSTER and ADUROSMART_SPECIFIC_CLUSTER two rawdata are reported at the same time, and LEVEL_CONTROL_CLUSTER will be reported multiple times,
2、Now， so the analysis of the button data of LEVEL_CONTROL_CLUSTER on the 2 or 3 button is removed, and only  parsed the data of ADUROSMART_SPECIFIC_CLUSTER on the 2 or 3 button, so that other engineer can get the single push event result of triggered by the once press button 2 or 3.

Automated-House commented:
Will this also fix the battery reporting? It currently shows 0% in the app since the device never reports it.

adurosmart commented:
@prjct92eh2  
Thank you for review.
Yes,fixed the battery reporting.when user pushed the button battery will reporting.

cgmckeever commented on devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy:
```diff
@@ -233,13 +246,48 @@ private getSupportedButtonValues() {
+		} else if (descMap.command == "00") {
+		    buttonNumber = 4
+		}
+	} else if (descMap.clusterInt == ADUROSMART_SPECIFIC_CLUSTER) {
```
This could possibly register dual events.

Commit:
Update zigbee-multi-button.groovy

Solve the problem of duplication of 1 and 4 button events in the dimming remote control

adurosmart commented:
@greens 
Can you help me to review and merge this code?
Thank you very much.

cgmckeever commented:
@adurosmart [report that this handler is still flakey](https://community.webcore.co/t/issues-with-adurosmart-eria-wireless-dimmer/17172/75?u=cgmckeever) ... Also, you wont get an approval on the code till conflicts are resolved 

`This branch has conflicts that must be resolved`
